packages/adapter-vercel/index.d.ts

```diff
@@ -98,6 +98,10 @@ export interface EdgeConfig {
 	 * If `true`, this route will always be deployed as its own separate function
 	 */
 	split?: boolean;
+	/**
+	 * https://vercel.com/docs/build-output-api/v3/configuration#images
+	 */
+	images?: ImagesConfig;
```

Comment:
this (and the equivalent option in `ServerlessConfig` is wrong — `images` is project-level config, not route-level config. I should have looked at #8667 more closely, we're going to need to revert that part of it and that means an annoying breaking change

Reply:
Given that it's a bug, do we really need to make it a breaking change? I don't think that option even worked when added at the route config level

Reply:
True, the fact that it never worked probably makes it acceptable, so I could be talked out of it. I just tend to be fairly liberal with majors for things like adapters

Reply:
Even if the current configuration doesn't do anything, is it possible that moving this could start to make someone's project stop building because of type checking?

Reply:
If they had something like this, then yes:

```js
// +page.server.js
import type { ServerlessConfig } from '@sveltejs/adapter-vercel';

export const config: ServerlessConfig = {
  images: {}
};
```

The gamble is that no-one _would_ have done that, because it doesn't do anything

Reply:
I'm inclined to make that gamble 😅